ni_measurementlink_generator/ni_measurementlink_generator/template.py

```diff
@@ -10,7 +10,7 @@
 def _render_template(template_name: str, **template_args) -> str:
     file_path = str(pathlib.Path(__file__).parent / "templates" / template_name)
 
-    with open(file_path, "r") as file:
+    with open(file_path, "r",encoding='UTF-8') as file:
```

Comment:
This is correct, but it highlights the fact that this script reads the template rather than letting Mako do it.

Could you please test this version of the fix and verify that it still fixes the problem? https://github.com/ni/measurementlink-python/commit/a111c1941d5d8c383566d858abf9d14dd6f39abe

If so, I will post a follow-up PR with this change as well as some error handling fixes.

Reply:
I've test your commit. Yes it could work. I just let you post a follow-up PR.